### PR TITLE
[SRVCOM 1299] Serverless Installing CLI add direct links to kn cli

### PR DESCRIPTION
**Pull request by SNiemann15**
OCP version for cherry-picking: 4.6, 4.7, 4.8
JIRA issue: https://issues.redhat.com/browse/SRVCOM-1299
Reviewer: Tony Gargya for IBM P/Z
Preview pages: https://deploy-preview-32169--osdocs.netlify.app/openshift-enterprise/latest/serverless/installing-kn.html

### CODE REVIEW

**Comment by dsimansk** [comment]
@SNiemann15 to followup our yesterday's chat. Indeed the `tar` command is working on Windows 10 build xyz, it's more than 2 years, therefore we can assume customers are on it.

The following works on Windows command line as well:
```
tar -xf kn-windows-amd64.zip
```


>Hello, the point is that we have to differentiate between downloading and installing. There are several ways to download the archive for different operating systems and for each a way to unpack and install it. Can we just point out that it is possible to download the archive via the web console and installing is os specific?

The point Tobias raised is indeed a good one, but I guess that's a bigger change, not necessarily needed right away, to restructure the docs.

**Comment by SNiemann15** [comment]
> @SNiemann15 to followup our yesterday's chat. Indeed the `tar` command is working on Windows 10 build xyz, it's more than 2 years, therefore we can assume customers are on it.
> 
> The following works on Windows command line as well:
> 
> ```
> tar -xf kn-windows-amd64.zip
> ```
> 
> > Hello, the point is that we have to differentiate between downloading and installing. There are several ways to download the archive for different operating systems and for each a way to unpack and install it. Can we just point out that it is possible to download the archive via the web console and installing is os specific?
> 
> The point Tobias raised is indeed a good one, but I guess that's a bigger change, not necessarily needed right away, to restructure the docs.

@abrennan89 Maybe something to think about for a future release.

File: modules/serverless-installing-cli-web-console.adoc [hunk 1]
```diff
@@ -3,9 +3,9 @@
 // serverless/knative-client.adoc
 
 [id="installing-cli-web-console_{context}"]
-= Installing the kn CLI using the {product-title} web console
+= Installing the Knative CLI (`kn`) using the {product-title} web console
 
-Once the {ServerlessOperatorName} is installed, you will see a link to download the `kn` CLI for Linux, macOS and Windows from the *Command Line Tools* page in the {product-title} web console.
+Once the {ServerlessOperatorName} is installed, you will see a link to download the `kn` CLI for Linux (x86_64, amd64, s390x, ppc64le), macOS, and Windows from the *Command Line Tools* page in the {product-title} web console.
```
Comment (abrennan89) [inline modules/serverless-installing-cli-web-console.adoc:8]:
```suggestion
Once the {ServerlessOperatorName} is installed, you will see a link to download the `kn` CLI for Linux (x86_64, amd64, s390x, ppc64le), macOS, or Windows, from the *Command Line Tools* page in the {product-title} web console.
```

**Review by abrennan89 — COMMENTED** [review]
One small suggestion otherwise lgtm.
Any larger restructure changes for future releases should be opened as Jira issues in our backlog. Thanks.

**Comment by maschmid** [comment]
QE approved

File: modules/serverless-installing-cli-linux-ibm-power-rpm.adoc [hunk 2]
```diff
@@ -3,7 +3,7 @@
 // * serverless/installing-kn.adoc
 
 [id="installing-cli-linux-ibm-power-rpm_{context}"]
-= Installing the Knative CLI for Linux on IBM Power Systems using an RPM
+= Installing the Knative CLI (`kn`) for Linux on IBM Power Systems using an RPM
```
Comment (kalexand-rh) [inline modules/serverless-installing-cli-linux-ibm-power-rpm.adoc:6]: You need to remove the backticks in all the headings based on the  [repo guidelines](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#assemblymodule-titles-and-section-headings):

> Do not use backticks or other markup in assembly or module headings.

Personally, I'd remove the abbreviation from the heading and add the expanded form and explanation to the first line of text. The ISG says:

> Avoid using abbreviations in headings and titles unless an abbreviation is well known or the spelled-out form is too long.

You're using an acceptable pattern, but check out the rest of the guidance in the "Headings and titles" section.

Whatever changes you make, please be consistent across the PR. :)

**Review by kalexand-rh — COMMENTED** [review]
There's one style issues that shows up a couple of places, but it otherwise LGTM!

**Pull request merged**